cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -68,6 +68,8 @@ else()
   set(patch_cmd "@CMAKE_COMMAND@" -E echo "Dummy patch command")
 endif()
 
+string(COMPARE EQUAL "@CMAKE_CXX_COMPILER_ID@" "Clang" compiler_is_clang)
```

Comment:
Build scheme changed, please update `PACKAGE_INTERNAL_DEPS_ID` of all `hunter.cmake` files from [cmake/projects/Boost](https://github.com/ruslo/hunter/tree/master/cmake/projects/Boost) directory.

Reply:
That was the missing piece. Thanks!